### PR TITLE
Load tweets on client-side

### LINKED ISSUE #15: Render tweets on client-side instead of build-time
## Guide
https://www.d-hagemeier.com/en/import-tweets-from-twitter-api-in-11ty#h-create-build-hook

## Uses
- IFTTT (or equivalent)
- Netlify build hooks

### PR DESCRIPTION
**Pull request by nayaabkhan**
Resolves #15

### CODE REVIEW

File: netlify/functions/tweets.cjs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  screen_name: 'AiryxOS',
+  count: 10,
+  exclude_replies: true,
+  include_rts: false,
+  tweet_mode: 'extended',
+}
+
+exports.handler = async function () {
+  try {
+    const timeline = await client.get('statuses/user_timeline', params)
```
Comment (nayaabkhan) [inline netlify/functions/tweets.cjs:20]: **Caveat:** Serverless functions by nature are stateless so expect more requests to the Twitter API than before.

File: netlify/functions/tweets.cjs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const Twitter = require('twitter')
+require('dotenv').config()
+
+const client = new Twitter({
+  consumer_key: process.env.TWITTER_CONSUMER_KEY,
+  consumer_secret: process.env.TWITTER_CONSUMER_SECRET,
+  bearer_token: process.env.TWITTER_BEARER_TOKEN,
+})
+
+const params = {
```
Comment (nayaabkhan) [inline netlify/functions/tweets.cjs:10]: **Note:** Some of these could be URL-parameterized. I'm curbing my urge to prematurely optimize 😅

File: netlify/functions/tweets.cjs (hunk 1 above)
Comment (mszoek) [reply netlify/functions/tweets.cjs:20]: Understood! Should be OK for now - our limit on the API is reasonable and we have caching through the CDN.

**Review by mszoek — APPROVED** [review]
LGTM!

**Pull request merged**